doc/newsboat.asciidoc

```diff
@@ -686,6 +686,23 @@ See also <<miniflux-tokenfile,`miniflux-tokenfile`>>, and
 
 Unlike with other backends, Miniflux's starring/bookmarking feature is not
 currently supported in Newsboat.
+In addition, Miniflux provides the ability to "star"
```

Comment:
I guess the previous sentence should be removed, because now the feature *is* supported.

Reply:
will do